portal/component-tests/partials/primary-navigation.component-test.js

```diff
@@ -1,50 +1,86 @@
-const { ADMIN, MAKER, CHECKER } = require('../../server/constants/roles');
-const { NON_MAKER_OR_CHECKER_OR_ADMIN_ROLES } = require('../../test-helpers/common-role-lists');
+const ROLES = require('../../server/constants/roles');
```

Comment:
Minor change but maybe this could be replaced with `const { ROLES } = require('../../server/constants')`